### ISSUE TITLE
color scale sizing

### ISSUE THREAD

**Issue by davism84** [monarch-initiative/phenogrid#180]
color scale doesn't look good when grid is small, in inverted mode.  consider using a fixed size, if overall grid width is below a certain width

![image](https://cloud.githubusercontent.com/assets/10049022/9964222/16ed7dc8-5dfd-11e5-9b59-28ed14cdb80a.png)

**Comment by yuanzhou** [comment]
We need your opinion @harryhoch , we even tried to place the gradient legend down to the overview map (this way we don't need to worry about the scaling issue). But it makes that area too tight and also leaves an empty space at the bottom between unmatched and options button.

Do you think that we can simply hide the "Invert Axis" option in the menu if there are less than 10 (or a different number) phenotypes? So users don't bother with the axis flipping?

**Comment by harryhoch** [comment]
What if we put the legend right under the "phenotype similarity comparison" label? - ie, at the top?

**Comment by yuanzhou** [comment]
Thanks @harryhoch size-wise, it sure will work. I've made changes to always show the top title's full length regardless of the number of rows/columns or if inverting axis. So moving the gradient legend under the top title should work.

**Comment by harryhoch** [comment]
great. thanks.

**Comment by yuanzhou** [comment]
Tried to move it to the top of grid and under the main title, didn't look good due to the alignment. So I came up with a better idea. Made the gradient legend thinner and shorter, and moved it down a little bit. Now it looks more like a legend and doesn't matter if we invert the axis.

![capture](https://cloud.githubusercontent.com/assets/195873/10076887/b342afce-62ab-11e5-8398-39d9fac366c3.PNG)

![capture2](https://cloud.githubusercontent.com/assets/195873/10076891/b57089e2-62ab-11e5-8a3c-c37c491e1e7b.PNG)
